ui/src/app/workflows/components/workflow-logs-viewer/workflow-logs-viewer.tsx

```diff
@@ -18,16 +18,21 @@ interface WorkflowLogsViewerProps {
 interface WorkflowLogsViewerState {
     error?: Error;
     loaded: boolean;
-    lines: string[];
+    lineCount: number;
+}
+
+function identity<T>(value: T) {
```

Comment:
Do you have a preferred place to put helper function like this?  I see there's a `utils` file that exports a giant object of helpers, however I wanted to keep this as a separate function for minification/tree shaking purposes and wasn't sure that was the right location.

Reply:
I don't feel strongly about this.